src/quick-lint-js/diag/diagnostic-types-2.h

```diff
@@ -3261,6 +3261,26 @@ struct Diag_Missing_Comma_Between_Array_Elements {
                   ARG(expected_comma))]]  //
   Source_Code_Span expected_comma;
 };
+
+struct Diag_Class_Generator_On_Getter_Or_Setter {
+  [[qljs::diag("E0713", Diagnostic_Severity::error)]]  //
+  [[qljs::message("'*' keyword is not allowed on getters or setters",
+                  ARG(generator_keyword))]]                    //
```

Comment:
I think this is a confusing message. "'*' keyword" doesn't make sense because `*` is not a keyword.

I think a better phrasing is: "getters and setters cannot be generators"